packages/collector/test/tracing/database/redis/test.js

```diff
@@ -22,7 +22,7 @@ const ProcessControls = require('../../../test_util/ProcessControls');
 const globalAgent = require('../../../globalAgent');
 
 describe('tracing/redis', function () {
-  ['latest', 0].forEach(redisVersion => {
+  ['latest', 'v3', 'v0'].forEach(redisVersion => {
```

Comment:
@kirrg001 Just saw that (after submitting my review) - looks like we might need to disable tests for `latest` on Node.js 10. The 4.0 [changelog](https://github.com/redis/node-redis/blob/master/CHANGELOG.md#breaking-changes) says Node.js 12 is the minimum version now.

Reply:
Good spot 🙏